test/e2e/crud.go

```diff
@@ -23,15 +23,29 @@ import (
 
 	kubeclientset "k8s.io/client-go/kubernetes"
 	"k8s.io/federation/pkg/federatedtypes"
+	"k8s.io/federation/test/common"
 	fedframework "k8s.io/federation/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+var _ = framework.KubeDescribe("Federated namespace [Feature:Federation][Experimental]", func() {
+	f := fedframework.NewDefaultFederatedFramework("federated-namespace")
+	Describe(fmt.Sprintf("Federated namespace resources"), func() {
+		It("should create, read, update and delete successfully", func() {
+			// Load clients only if not skipping to avoid doing
```

Comment:
This comment appears unnecessary since there is no conditional skip invocation.  Now that federation is out of tree, skipping likely doesn't make sense anyway since any e2e run in k/f is a federation test run.

Reply:
Ok, I will remove it from the Federated types method (as well as the conditional it references) as well then.